charts/tidb-cluster/templates/tidb-cluster.yaml

```diff
@@ -2,6 +2,12 @@ apiVersion: pingcap.com/v1alpha1
 kind: TidbCluster
 metadata:
   name: {{ template "cluster.name" . }}
+{{- if .Values.enableConfigMapRollout }}
+  annotations:
+    pingcap.com/pd.{{ template "cluster.name" . }}-pd.sha: {{ template "pd-configmap.data-digest" . }}
```

Comment:
```
pingcap.com/pd.{{ template "cluster.name" . }}-pd.sha
```

There are two duplicate`pd`s in the annotations,  can we have only one?

```
pingcap.com/{{ template "cluster.name" . }}-pd.sha
```

Reply:
I suppose no, the form `pd.{{ template "cluster.name" . }}-pd.sha` is consists of `{component}.{configmap-name}.sha` according to the design document, and in this case, the `{configmap-name}` is just happen to be `{{ template "cluster.name" . }}-pd`, which have not yet been a constrain for the configmap name.